packages/contracts/src/2.0.0/protocol/Exchange/MixinExchangeCore.sol

```diff
@@ -86,43 +89,14 @@ contract MixinExchangeCore is
         bytes memory signature
     )
         public
```

Comment:
Can we promote it to `external` now?

Reply:
`external` doesn't work functions that have structs as arguments yet.